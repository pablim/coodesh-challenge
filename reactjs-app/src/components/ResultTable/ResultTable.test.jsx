import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { ResultTable } from '../'

describe('Reders tests', () => {
	test('Render ResultTable component', () => {
		const { container } = render(<ResultTable items={[]}/>);
		expect(container).toMatchSnapshot();
	});
})
