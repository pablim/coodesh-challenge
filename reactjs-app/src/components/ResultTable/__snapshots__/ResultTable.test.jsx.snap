// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reders tests Render ResultTable component 1`] = `
<div>
  <div
    class="w-100"
  >
    <table
      class="table table-hover table-striped table-dark"
    >
      <thead
        class="table-light"
      >
        <tr>
          <th>
            Seller
          </th>
          <th>
            Product
          </th>
          <th>
            Transaction
          </th>
          <th>
            Amount
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</div>
`;
